cli/connect/shortcuts.go

```diff
@@ -0,0 +1,29 @@
+package connect
+
+// getShortcutsList is a command for a get shortcuts and hotkeys list.
+const getShortcutsList = "\\shortcuts"
+
+const shortcutListText = `---
+- - |
+    Available hotkeys and shortcuts:
+
+       Ctrl + J / Ctrl + M [Enter] -- Enter the command
+       Ctrl + A [Home]             -- Go to the beginning of the line
```

Comment:
In multi-line command case this moves to the beginning of the command, not line. The same with `Ctrl+E`.
```suggestion
       Ctrl + A [Home]             -- Go to the beginning of the command
```

Reply:
fixed